ruby3.2-bouncy-castle-java.yaml

```diff
@@ -0,0 +1,60 @@
+# Generated from http://github.com/jruby/jruby/tree/master/gems/bouncy-castle-java/
+package:
+  name: ruby3.2-bouncy-castle-java
```

Comment:
Hoping @found-it can help here re: ruby :) particularly the package naming, is this convention? We're going to ruby3.3 soon so any concerns here?